### DISCUSSION TITLE
Validation rule date_format not working as expected with RFC3339

### DISCUSSION THREAD

**Discussion by hullabaloooooooo** [laravel/framework#46685] [Q&A]
- Laravel Version: 8.83.27
- PHP Version: 8.1.9

### Description:

All variants of present rules works not OK with RFC3339 datetimes.
```php
[
'date_format:' . DateTimeInterface::RFC3339,
'date_format:Y-m-d\\TH:i:sP',
'date_format:Y-m-d\TH:i:sP',
'date_format:"Y-m-d\\TH:i:sP"',
"date_format:Y-m-d\\TH:i:sP",
];
```

**Trying dates:**
2002-10-02T10:00:00-05:00
2002-10-02T15:00:00Z
2002-10-02T15:00:00.05Z
1985-04-12T23:20:50.52Z
1996-12-19T16:39:57-08:00
1990-12-31T23:59:60Z
1990-12-31T15:59:60-08:00
1937-01-01T12:00:27.87+00:20

**Works only:**
2002-10-02T10:00:00-05:00

**Comment by zackarydev** [comment]
Bump on this ?

**Reply by ghost** [reply]
it works also on
 date_format:Y-m-d 
 date_format:Y-m-d H:i:s
date_format:Y
date_format:Ym
date_format:Y-m-d 00:00:00
